k8s/aat/common-overlay/rd/kustomization.yaml

```diff
@@ -3,11 +3,11 @@ kind: Kustomization
 namespace: rd
 bases:
 - ../../../namespaces/rd
-- ../../../namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml
+# - ../../../namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml
```

Comment:
Why not files as well? Is this temporary ?

Reply:
Yes temporary to remove and re-add the config ones pods are removed.